orttraining/orttraining/core/graph/gradient_builder.cc

```diff
@@ -958,6 +962,58 @@ IMPLEMENT_GRADIENT_BUILDER(GetGeluGradient) {
               {GI(0)})};
 }
 
+namespace {
+std::vector<NodeDef> GetBiasGeluGradNodes(
+    bool use_approximation,
```

Comment:
>use_approximation [](start = 9, length = 17)

I would just call it is_fastgelu... 

both gelu and fastgelu is an approximation of the "GELU" function... maybe in the future, we will take fastgelu as the default approximation of  "GELU"

Reply:
hm, i was following existing conventions. e.g. the graph transformer GeluApproximation, and there's a bool template parameter named use_approximation in the CPU GELU kernels.

---
In reply to: [441038593](https://github.com/microsoft/onnxruntime/pull/4146#discussion_r441038593) [](ancestors = 441038593)

Reply:
will leave it as is for consistency

---
In reply to: [441155952](https://github.com/microsoft/onnxruntime/pull/4146#discussion_r441155952) [](ancestors = 441155952,441038593)